cmd/ethrex_l2/src/commands/utils.rs

```diff
@@ -0,0 +1,28 @@
+use clap::Subcommand;
+use keccak_hash::H256;
+use secp256k1::SecretKey;
+
+#[derive(Subcommand)]
+pub(crate) enum Command {
+    #[clap(about = "Convert private key to address.")]
```

Comment:
Let's change the name of the command so it's shorter. Here's a suggestion, but it might be better options
```suggestion
    #[clap(
        name = "address",
        visible_aliases = ["a"],
        about = "Convert private key to address."
    )]
```

Reply:
Nice! I like it